minio/compat.py

```diff
@@ -41,10 +41,10 @@
     queue_empty = Empty
 
     from urllib import quote
-    _urlencode = quote
+    _quote = quote
 
     from urllib import unquote
-    urldecode = unquote
+    unquote = unquote
```

Comment:
This change does not make sense here

Reply:
Eek - this diff makes that painfully obvious! I'll remove this line.

Reply:
I still see this @hardbyte. Can we name it as `_unquote`

Reply:
Done